.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,23 @@
+name: Release Drafter
+
+# Runs autolabeler on every PR
+# Creates/updates draft release on every push to master
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
I'd suggest adding `workflow_dispatch` to this workflow because you may sometimes want to go back and change the labels on a PR which has yet to be released

Reply:
sure thing